dev/run-tests-jenkins.py

```diff
@@ -183,6 +183,7 @@ def main():
     # Switch the Hive profile based on the PR title:
     if "test-hive1.2" in ghprb_pull_title:
         os.environ["AMPLAB_JENKINS_BUILD_HIVE_PROFILE"] = "hive1.2"
+        os.environ["AMPLAB_JENKINS_BUILD_PROFILE"] = "hadoop2.7"
```

Comment:
Let's don't do this but rather fail with an explicit error messages. It will be confusing what Hive 1.2 uses its Hadoop version.

Reply:
Ya. +1 for @HyukjinKwon 's advice.

Reply:
+1. thanks for the advice